<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yandex_lyceum</class>
 <widget class="QMainWindow" name="Yandex_lyceum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="welcome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добро пожаловать</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сообщения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Yandex_lyceum</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttons">
     <item>
      <widget class="QPushButton" name="incoming_message_button">
       <property name="text">
        <string>Входящие</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_message_button">
       <property name="text">
        <string>Отправленные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="spam_button">
       <property name="text">
        <string>Спам</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_message_button">
       <property name="text">
        <string>Написать новое</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_messages">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="read_message">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>331</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="new_message_group">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>341</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>331</width>
       <height>341</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="box">
      <item>
       <widget class="QLabel" name="user_error">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Выберите кому отправить письмо!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_error">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Введите название!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="message_error">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Нельзя отправить пустое сообщение!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Кому:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="list_users"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Название:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="title_message"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Тема:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="topic_message"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Сообщение:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="message"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
